sklearn/tests/test_multioutput.py

```diff
@@ -646,7 +646,7 @@ def fit(self, X, y, **fit_params):
             self.sample_weight_ = fit_params["sample_weight"]
             super().fit(X, y, **fit_params)
 
-    model = RegressorChain(MySGD())
+    model = RegressorChain(MySGD().set_fit_request(sample_weight=True))
```

Comment:
We need a test that checks that the existing behaviour still works but issues a `DeprecationWarning`.

Reply:
I think the common tests are supposed to do that.